src/Controller/Admin/BlogController.php

```diff
@@ -112,7 +112,7 @@ public function new(Request $request, EntityManagerInterface $entityManager): Re
     public function show(Post $post): Response
     {
         // This security check can also be performed
-        // using an annotation: @IsGranted("show", subject="post", message="Posts can only be shown to their authors.")
+        // using a PHP attribute: #[IsGranted("show", subject="post", message="Posts can only be shown to their authors.")]
```

Comment:
I think we should write params with `:` to match the same style in other spots, e.g. in https://github.com/symfony/demo/blob/f1ee7c6b8a70f8fed2ca032052a7760d58299cef/src/Controller/Admin/BlogController.php#L127

```suggestion
        // using a PHP attribute: #[IsGranted('show', subject: 'post', message: 'Posts can only be shown to their authors.')]
```

Reply:
For sure! Sorry for the oversight.